src/api/track.ts

```diff
@@ -160,33 +160,33 @@ class TrackAPI {
           sclVal,
           cslVal,
         };
+      }
```

Comment:
This is really the key change. Ensures the `return` is outside of `if (options.debug ...)`